server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -366,7 +366,7 @@ private void cleanupAndDelete(Owner owner, boolean revokeCerts) {
             }
             else {
                 // otherwise just remove them without touching the CRL
-                poolManager.removeAllEntitlements(c);
+                poolManager.revokeAllEntitlements(c);
```

Comment:
As @vritant correctly pointed out (@mstead this is also in the email I sent yesterday), this may not be very good decision. By doing this I effectively created code that ignores `revokeCerts` flag in method 
```java
 public void deleteOwner(@PathParam("owner_key") String ownerKey,
        @QueryParam("revoke") @DefaultValue("true") boolean revoke) 
```

After my change the method looks like this:
```java
     for (Consumer c : consumers) {
            log.info("Removing all entitlements for consumer: " + c);

            if (revokeCerts) {
                poolManager.revokeAllEntitlements(c);
            }
            else {
                // otherwise just remove them without touching the CRL
                poolManager.removeAllEntitlements(c);
            }
        }
```

So even in case when a REST client sets this flag to false, then code in this PR will regenerate certificate regardless. 

I should either
- clean up the resource method and remove the flag alltogether (which is API change)
- introduce the flag into my code, which is very easy thing to do (@vritant I think this is the best route to take right now, what do you think?)